### PR TITLE
First iteration of calendar

### PR DESCRIPTION
**Pull request by lutrarutra**
Created pages:
- /calendar (responsive)
- /edit/calendar (not small screen friendly)

Implementation summary:
- Races (templated from /static/json/gp_references.json) can be added to calendar. Same circuit can be added as many times as one wishes. 
- Time and date for each race are stored separately as text. 
- Calendar is stored in `races`-table with 4 columns: id (PK), gp_id (reference to GP from `statc/json/gp_reference.json`), date, and time
- DB is cached to /static/json/calendar.json.
- Cache is included in `.gitignore`

### CODE REVIEW

File: internal/handlers/admin.go [hunk 1]
```diff
@@ -15,22 +16,14 @@ func (m *Repository) EditTeams(w http.ResponseWriter, r *http.Request) {
...
-	driverNames := make(map[int]string)
-	var reserveDrivers []string
-	for _, driver := range templateData.Drivers {
-		if driver.TeamID == -1 {
-			reserveDrivers = append(reserveDrivers, driver.Name)
-		}
-		driverNames[driver.ID] = driver.Name
-	}
-
-	dataMap := make(map[string]interface{})
```
Comment (iMeisa) [inline internal/handlers/admin.go:27]: Non essential removal of functions

File: internal/models/templatedata.go [hunk 2]
```diff
@@ -4,23 +4,64 @@ import (
...
+
+	err = json.Unmarshal(rawFile, &races)
+	if err != nil {
+		log.Println(err)
+	}
+
+	sort.Slice(races, func(i, j int) bool {
+		return races[i].Date < races[j].Date
+	})
+
```
Comment (iMeisa) [inline internal/models/templatedata.go:41]: Sort will be done by SQL. (No changes for pull request)

File: templates/base.layout.tmpl [hunk 3]
```diff
@@ -51,6 +51,7 @@
                 <ul class="navbar-nav">
                     <li class="nav-item"><a class="nav-link" href="/standings" id="nav-standings">Standings</a></li>
                     <li class="nav-item"><a class="nav-link" href="/grid" id="nav-grid">Grid</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/calendar" id="nav-grid">Calendar</a></li>
```
Comment (iMeisa) [inline templates/base.layout.tmpl:54]: Change id to nav-calendar

**Review by iMeisa — CHANGES_REQUESTED** [review]
Couple of changes and then should be good for merge

File: internal/handlers/admin.go (hunk 1 above)
Comment (lutrarutra) [reply internal/handlers/admin.go:27]: Done in [f4491f1](https://github.com/iMeisa/serserilig.meisa.xyz/pull/2/commits/f4491f1f1396444f89e6ab4ff28365876491d598)

**Review by iMeisa — APPROVED** [review]

**Pull request merged**